package.json

```diff
@@ -446,6 +446,12 @@
         "title": "View Flink Statement Results",
         "category": "Confluent: Flink Statements"
       },
+      {
+        "command": "confluent.handleUri",
+        "title": "Handle URI",
+        "icon": "$(link)",
+        "category": "Confluent"
+      },
```

Comment:
Should we enable the command only if the env var `CONFLUENT_VSCODE_E2E_TESTING` is set, assuming it doesn't provide value to _regular_ users?